hamocc/mo_biomod.F90

```diff
@@ -4,359 +4,345 @@
+  real, dimension (:,:),   allocatable, public :: expoca
+  real, dimension (:,:),   allocatable, public :: exposi
+  real, dimension (:,:),   allocatable, public :: intphosy
+  real, dimension (:,:),   allocatable, public :: intdnit
+  real, dimension (:,:),   allocatable, public :: intnfix
+  real, dimension (:,:),   allocatable, public :: intdmsprod
+  real, dimension (:,:),   allocatable, public :: intdms_bac
+  real, dimension (:,:),   allocatable, public :: intdms_uv
+  real, dimension (:,:),   allocatable, public :: carflx0100
+  real, dimension (:,:),   allocatable, public :: carflx0500
+  real, dimension (:,:),   allocatable, public :: carflx1000
+  real, dimension (:,:),   allocatable, public :: carflx2000
+  real, dimension (:,:),   allocatable, public :: carflx4000
+  real, dimension (:,:),   allocatable, public :: carflx_bot
+  real, dimension (:,:),   allocatable, public :: bsiflx0100
+  real, dimension (:,:),   allocatable, public :: bsiflx0500
+  real, dimension (:,:),   allocatable, public :: bsiflx1000
+  real, dimension (:,:),   allocatable, public :: bsiflx2000
+  real, dimension (:,:),   allocatable, public :: bsiflx4000
+  real, dimension (:,:),   allocatable, public :: bsiflx_bot
+  real, dimension (:,:),   allocatable, public :: calflx0100
+  real, dimension (:,:),   allocatable, public :: calflx0500
+  real, dimension (:,:),   allocatable, public :: calflx1000
+  real, dimension (:,:),   allocatable, public :: calflx2000
+  real, dimension (:,:),   allocatable, public :: calflx4000
+  real, dimension (:,:),   allocatable, public :: calflx_bot
+  real, dimension (:,:,:), allocatable, public :: phosy3d
+
+  ! Variables for interactive phytoplanktion absorption (use_FB_BGC_OCE=.true.)
+  real, dimension (:,:,:), allocatable, public :: abs_oce
+
+  ! Variables for aggregation scheme (use_AGG=.true.)
+  real, dimension (:,:,:), allocatable, public  :: wmass
+  real, dimension (:,:,:), allocatable, public  :: wnumb
+  real, dimension (:,:,:), allocatable, public  :: eps3d
+  real, dimension (:,:,:), allocatable, public  :: asize3d
+
+  ! Variables for bromoform scheme (use_BROMO=.true.)
+  real, dimension (:,:),   allocatable, public  :: int_chbr3_prod
+  real, dimension (:,:),   allocatable, public  :: int_chbr3_uv
+
+  real, public :: growth_co2
+  real, public :: bifr13_perm
+
+CONTAINS
+
+  subroutine alloc_mem_biomod(kpie,kpje,kpke)
+    !******************************************************************************
+    ! ALLOC_MEM_BIOMOD - Allocate variables in this module
+    !******************************************************************************
+    use mod_xc,         only: mnproc
+    use mo_control_bgc, only: io_stdo_bgc
+    use mo_control_bgc, only: use_FB_BGC_OCE,use_AGG,use_BROMO
+
+    ! Arguments
+    integer, intent(in) :: kpie
+    integer, intent(in) :: kpje
+    integer, intent(in) :: kpke
+
+    ! local variables
+    integer :: errstat
+
+    if (mnproc.eq.1) THEN
```

Comment:
The `THEN` (here and thereafter) could be also written in small letters.

Reply:
I have changed all THEN->then and also addressed your other review comments.